Doc/library/sqlite3.rst

```diff
@@ -478,9 +478,9 @@ Module constants
 
    .. note::
 
-      The :mod:`!sqlite3` module supports both ``qmark`` and ``numeric`` DB-API
-      parameter styles, because that is what the underlying SQLite library
-      supports. However, the DB-API does not allow multiple values for
+      The :mod:`!sqlite3` module supports ``qmark``, ``numeric``, and ``named``
+      DB-API parameter styles, because that is what the underlying SQLite
+      library supports. However, the DB-API does not allow multiple values for
```

Comment:
Reflowing bco. the 79 char line limit.

```suggestion
      The :mod:`!sqlite3` module supports ``qmark``, ``numeric``,
      and ``named`` DB-API parameter styles,
      because that is what the underlying SQLite library supports.
      However, the DB-API does not allow multiple values for
```